### ISSUE TITLE
🐝 [Bug]The Stone Watcher (Alliance)

### ISSUE THREAD

**Issue by ShiyoKozuki** [vmangos/core#2376]
## 🐝 Bug report - Quest
![image](https://github.com/vmangos/core/assets/58101246/19641d46-33be-42f2-9efa-26a00cf6b05d)
![image](https://github.com/vmangos/core/assets/58101246/ffea71e8-863e-4ef7-9836-7b91e7c23fc8)

Alliance is elegible for the Horde version of this quest (Return to ThunderBluff). Abandoning the horde version also forces you to toss the discs you need to complete the Alliance version (Return to IronForge) which can make someone have to travel all the way to the bottom of Tanaris again.

### Expected behavior
Alliance shouldn't be eligible for horde quests.

### Version & Environment

Client Version:  1.12.1

Commit Hash:  newest

OS Client:  win10
OS Server:  win10

### Crashlog

- None

**Comment by NickTyrer** [comment]
Closed here #1970

**Comment by ShiyoKozuki** [comment]
> Closed here #1970

This commit fixes it then and can close this issue?

**Comment by mserajnik** [comment]
https://github.com/vmangos/core/pull/1970/commits/37fc4d04ac1a4689d5466b94759821c3a2b315ec fixes it, which is part of #1970 but hasn't been merged yet.

I am sure Nick will remember to close this issue once the commit gets merged (as he has done with other issues that got closed by him).

If you want to manually apply the fix in the meantime you can run the following:

```sql
UPDATE `quest_template` SET `RequiredRaces` = 77 WHERE `entry` = 2977;
UPDATE `quest_template` SET `RequiredRaces` = 178 WHERE `entry` = 2967;
```
